frontend/src/pages/UpdateUser.tsx

```diff
@@ -1,5 +1,119 @@
-import { Box } from '@mui/material';
+import {
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+  Button,
+  FormControl,
+} from '@mui/material';
+import Background from '../components/Background';
+import { useState } from 'react';
 
 export const UpdateUser = () => {
-  return <Box>This is the UpdateUser Page!</Box>;
+  const [userDetails, setUserDetails] = useState<{
+    emojis: string;
+    faculty: string;
```

Comment:
A Faculty is a string but there's only a finite number of strings that it can be e.g. it can't be 'testing'. When we have a finite number of strings it can be, we can use Typescript to make it stricter and reduce the chances of errors.

Define a types.ts file and define an enum for faculty. You can copy how it's done on the backend (see [here](https://github.com/Siyer2/uni-connector/blob/0e1feb4ae5ff42206da014c088bbe2a770ee8cd3/backend/src/user/types.ts#L12))

Now we can change out state's faculty type from string to Faculty. Instead of doing: 
```
<MenuItem value={'Arts, Design & Architecture'}>
              Arts, Design & Architecture
</MenuItem>
```
we do
```
<MenuItem value={Faculty.ArtsDesignAndArchitecture}>
              Arts, Design & Architecture
</MenuItem>
```

What's the benefit of doing this? We're going to have this update user's page and we're also going to have a page where a user can change their faculty later (e.g. maybe 1 year later, they transfer to another faculty). Imagine if we had to type out 'Arts, Design & Architecture' on both pages. Even worse, what happens if, accidentally, on 1 page it's 'Arts, Design & Architecture' and on another it's 'Arts & Architecture'. Our system will treat them as different faculties and cause issues. So we use enums to simplify this.